source/extensions/omni.isaac.lab/omni/isaac/lab/controllers/operational_space.py

```diff
@@ -185,43 +134,83 @@ def reset_idx(self, robot_ids: torch.Tensor = None):
         """Reset the internals."""
         pass
 
-    def set_command(self, command: torch.Tensor):
+    def set_command(self, command: torch.Tensor, ee_pose: torch.Tensor | None = None):
         """Set target end-effector pose or force command.
 
         Args:
             command: The target end-effector pose or force command.
```

Comment:
Can you expand this docstring to denote how the different impedance modes affect command shape and contents?

Reply:
I have edited the docstring as the following:

```
 """Set the task-space targets and impedance parameters.

        Args:
            command (torch.Tensor): A concatenated tensor of shape ('num_envs', 'action_dim') containing task-space
                targets (i.e., pose/wrench) and impedance parameters.
            current_ee_pose (torch.Tensor, optional): Current end-effector pose of shape ('num_envs', 7), containing position
                and quaternion (w, x, y, z). Required for relative commands. Defaults to None.

        Format:
            Task-space targets, ordered according to 'command_types':

                Absolute pose: shape ('num_envs', 7), containing position and quaternion (w, x, y, z).
                Relative pose: shape ('num_envs', 6), containing delta position and rotation in axis-angle form.
                Absolute wrench: shape ('num_envs', 6), containing force and torque.

            Impedance parameters: stiffness for 'variable_kp', or stiffness, followed by damping ratio for 'variable':

                Stiffness: shape ('num_envs', 6)
                Damping ratio: shape ('num_envs', 6)

        """
```